java.js

```diff
@@ -0,0 +1,82 @@
+const Store = {
+  getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  },
+
+  addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  },
+
+  removeBook(author) {
+    const books = Store.getBooks();
+    let index = 0;
+    books.filter((book) => {
+      if (book.author !== author) { index = +1; }
+      return books;
+    });
+    books.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+  },
+};
```

Comment:
- you did a great job by using the local storage however  the data are not preserved in the browser's memory when refreshing the page all books are desperate, please fix that by using correctly the localstorage  :+1: